lib/newgistics/update_shipment.rb

```diff
@@ -0,0 +1,27 @@
+module Newgistics
+  class UpdateShipment
+    include Virtus.model
+
+    attribute :id, String
+    attribute :order_id, String
+    attribute :add_items, Array[Item]
```

Comment:
Do we need to set a default value for this attribute? Maybe default to an empty array like we do with errors and warnings?